src/shared/efi-loader.c

```diff
@@ -61,18 +61,18 @@ int efi_loader_get_boot_usec(usec_t *ret_firmware, usec_t *ret_loader) {
         return 0;
 }
 
-int efi_loader_get_device_part_uuid(sd_id128_t *ret) {
-        _cleanup_free_ char *p = NULL;
+static int get_device_part_uuid(const char *variable, sd_id128_t *ret) {
```

Comment:
This is quite generic. Maybe better to name `efi_get_variable_uuid()` or so and move it to efivars.c?